### PR TITLE
fix: some styles

### PR DESCRIPTION
**Pull request by pregnantboy**
fix some styles mismatch

### standardize guide and what's new buttons
Screenshot 2023-08-29 at 7 38 28 PM

### z-index fix
Screenshot 2023-08-29 at 7 38 33 PM

### Fix search bar and button height mismatch

Screenshot 2023-08-29 at 7 33 35 PM

### CODE REVIEW

File: packages/frontend/src/components/AppBar/index.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default function AppBar(props: AppBarProps): React.ReactElement {
     <MuiAppBar
       color="transparent"
       position={drawerOpen && matchSmallScreens ? 'fixed' : 'relative'}
-      sx={{ zIndex: 1 }}
+      sx={{ zIndex: 11 }}
```
Comment (m0nggh) [inline packages/frontend/src/components/AppBar/index.tsx:50]: v small thing but can i ask why you chose this number for the zindex ah

File: packages/frontend/src/components/NewsDrawer/index.tsx [hunk 2]
```diff
@@ -45,26 +45,21 @@ export default function NewsDrawer() {
...
+            borderRadius="50%"
+            bg="#C05050"
+            w={1.5}
+            h={1.5}
+            top={2}
+            right={-1}
+            position="absolute"
+          />
+        )}
       </Button>
```
Comment (m0nggh) [inline packages/frontend/src/components/NewsDrawer/index.tsx:61]: is absolute positioning preferred over negative margins?

**Review by m0nggh — APPROVED** [review]
lgtm, thks for fixing the z-index issue, didn't catch that

File: packages/frontend/src/components/AppBar/index.tsx (hunk 1 above)
Comment (pregnantboy) [reply packages/frontend/src/components/AppBar/index.tsx:50]: it has to be higher than drawer

File: packages/frontend/src/components/NewsDrawer/index.tsx (hunk 2 above)
Comment (pregnantboy) [reply packages/frontend/src/components/NewsDrawer/index.tsx:61]: the button shifts with and without the Box if it's not position absolute

**Pull request merged**